frontend/tailwind.config.js

```diff
@@ -1,8 +1,20 @@
 /** @type {import('tailwindcss').Config} */
 export default {
   content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  darkMode: 'media',
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        dark: {
+          background: {
+            DEFAULT: 'var(--color-dark-background)',
+            card: {
+              textfield: 'var(--color-dark-textfield-card)',
+            },
```

Comment:
kindly make the changes as commented

Reply:
@krishnaacharyaa same here is this issue also resolved